scripts/colmap2nerf.py

```diff
@@ -333,6 +348,23 @@ def closest_point_2_lines(oa, da, ob, db): # returns point closest to both rays
 					up += c2w[0:3,1]
 
 				frame = {"file_path":name,"sharpness":b,"transform_matrix": c2w}
+				if len(cameras) != 1:
+					camera = cameras[int(elems[8])]
+					frame["camera_angle_x"] = camera["angle_x"]
```

Comment:
This could be expressed more concisely using `frame |= camera` if the `angle_x` parameter is renamed to `camera_angle_x`.